_data/chains/eip155-167007.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Taiko Jolnir L2",
+  "chain": "ETH",
+  "status": "incubating",
+  "icon": "taiko",
+  "rpc": ["https://rpc.jolnir.taiko.xyz"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "infoURL": "https://taiko.xyz",
+  "shortName": "tko-jolnir",
+  "chainId": 167007,
+  "networkId": 167007,
+  "explorers": [
+    {
+      "name": "blockscout",
+      "url": "https://explorer.jolnir.taiko.xyz",
```

Comment:
cannot be reached

Reply:
Block explorer is not live yet. I will set this pull request as a draft until it is live.